rules/windows/defense_evasion_network_connection_from_windows_binary.toml

```
@@ -1,6 +1,6 @@
 [metadata]
 creation_date = "2020/09/02"
-integration = ["endpoint", "windows"]
+integration = ["endpoint", "windows", "sentinel_one_cloud_funnel", "crowdstrike"]
```

```suggestion
integration = ["endpoint", "windows", "sentinel_one_cloud_funnel"]
```
CS is inconsistent with process.entity_id for now, network events don't have it yet (this is documented and pending fix), and it don't populate process.parent.executable